src/Table/Table.js

```diff
@@ -474,18 +368,5 @@ Table.propTypes = {
   sortBy: PropTypes.shape({
     order: PropTypes.oneOf(['asc', 'desc']),
     prop: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
-  }),
-
-  // Optional property to add transitions or turn them off. Default is off.
-  // Only available for tables that does not scroll
-  transition: PropTypes.bool,
```

Comment:
All of these props are useless now that we don't use `Gemini`. Transition does not work with `VirtualList` yet